python/ray/tests/test_runtime_env.py

```diff
@@ -43,7 +43,7 @@
 def test_get_wheel_filename():
     """Test the code that generates the filenames of the `latest` wheels."""
     # NOTE: These should not be changed for releases.
-    ray_version = "3.0.0.dev0"
+    ray_version = "2.7.0"
```

Comment:
these are failing the unit tests.

the note says that these should not be changed for releases.

Reply:
Will fix it